tests/test_wave_01.py

```diff
@@ -142,13 +142,13 @@ def test_moves_movie_from_watchlist_to_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
-    
-    raise Exception("Test needs to be completed.")
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
+    assert updated_data["watched"][1]["title"] == MOVIE_TITLE_1
+    assert updated_data["watched"][0]["title"] == "The Lord of the Functions: The Two Parameters"
```

Comment:
The spec for the function watch_movie doesn't state how movies are added to the watched list, so it's possible that it could add it to the front of the list rather than the end. This test requires that the movie be appended to the end of the list. Changing this to an in test would make the test work for different types of implementations:
```suggestion
    assert movie_to_watch in updated_data["watched"]
```